packages/block/src/blockConstructor.ts

```diff
@@ -0,0 +1,442 @@
+import { RLP } from '@ethereumjs/rlp'
```

Comment:
I would like to establish a general naming convention for this file name that we re-use throughout all libraries, so that it's just clear where to look for, same a bit as our `types.ts` files.

Can we take `constructors.ts` here in this case? That would be my pledge! 🙂 

So without `block` (redundant, I would also pledge for just always having one file for this per library), and also using plural (`constructors`), since several constructors (let's always use this name, and not fall back to singular if for the moment there is only one constructor present for a library).

Reply:
(so these comments go more or less 1:1 for the other PR for blockchain)